src/components/libvirtSlate.jsx

```diff
@@ -60,10 +61,19 @@ class LibvirtSlate extends React.Component {
     }
 
     startService() {
+        this.setState({ actionInProgress: true });
         const service = this.props.libvirtService;
 
-        enableLibvirt({ enable: this.state.libvirtEnabled, serviceName: service.name });
-        startLibvirt({ serviceName: service.name });
+        enableLibvirt({
+            enable: this.state.libvirtEnabled,
+            serviceName: service.name
+        })
+                .then(() => {
+                    startLibvirt({
```

Comment:
Big +1 on this serialization -- enable() does a daemon-reload, which tends to lock up with pending start/stop transactions. (I've spent a lot of time debugging this with pcp)